designer/server/src/main/scala/pl/touk/nussknacker/ui/notifications/NotificationService.scala

```diff
@@ -107,4 +113,10 @@ class NotificationServiceImpl(
     } yield notificationsForScenarioActivities
   }
 
+  private def fetchGlobalNotificationsAndTriggerEviction(limit: Instant) = {
+    val globalNotifications = globalNotificationRepository.fetchEntries(limit)
```

Comment:
right now if we have multiple refreshes previously to an FE client being lanched, the FE will have N notifications to refresh config, which will result in N calls to api for definition of processing type. It could either be solved by just sending the single latest notification of reload or by flattening them on FE into one request.

Reply:
I found the resolution of this problem on FE side more suitable. Take a look